### PR TITLE
Improve clientarch 

### PR DESCRIPTION
**Pull request by anacso17**

### CODE REVIEW

**Review by murilobalves — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]
very nice @anacso17!
I specially enjoyed the work on the `Time` class! 
Now we don't have to worry about time zones when providing the information, right?

File: siriuspy/siriuspy/clientarch/client.py [hunk 1]
```diff
@@ -44,77 +47,92 @@ def login(self, username, password):
             url, headers=headers, data=payload, verify=False)
         return b"authenticated" in response.content
 
-    def getPVsInfo(self, pvnames):
+    def getPVsInfo(self, pvnames, timeout=_TIMEOUT):
```
[the remark this answers is not in the document]
Comment (xresende) [reply siriuspy/siriuspy/clientarch/client.py:50]: it woud be cleaner. the price to pay is that, if we want to allow for different timeout values in class methods, a setter and getter for the class attribute should be implemented.

**Review by xresende — APPROVED** [review]

**Comment by anacso17** [comment]
@fernandohds564 exactly! Now, the get_iso8601handles timezone information!

File: siriuspy/siriuspy/clientarch/client.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (fernandohds564) [reply siriuspy/siriuspy/clientarch/client.py:50]: ohhh, right! I forgot about the class methods... maybe for those methods the argument could continue existing. What do you think @xresende ?

[the remark this answers is not in the document]
Comment (fernandohds564) [reply siriuspy/siriuspy/clientarch/client.py:50]: @xresende, I took a look and didn't find any class method in this class.

[the remark this answers is not in the document]
Comment (anacso17) [reply siriuspy/siriuspy/clientarch/client.py:50]: When I thought about this implementation, I considered that the user might want to set a different timeout for each method, or even change the timeout for each call, so I kept this syntax. If it is an attribute of the object, it must be set before each call, if this is the case. Which implementation do you think would be most appropriate?

[the remark this answers is not in the document]
Comment (fernandohds564) [reply siriuspy/siriuspy/clientarch/client.py:50]: I still prefer a cleaner signature for the methods. I don't think this use, changing the timeout at every call, will be very common.

[the remark this answers is not in the document]
Comment (anacso17) [reply siriuspy/siriuspy/clientarch/client.py]: done!

**Review by fernandohds564 — APPROVED** [review]

**Pull request merged**